WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1482,9 +1482,11 @@ DLEditViewContent           = padding:0;
 DLEditViewInput             = @GroupEditViewInput@;
 DLEditViewTabText           = padding-left:@ContactViewWrapperPadding@; padding-right:@ContactViewWrapperPadding@;
 DLEditViewTabPage           = @GroupEditViewContent@;
-DLPropertiesView            = 
-DLPropertiesViewRowLabel    = padding: 0 0 @SkinWrapperPadding@ 0;
-DLPropertiesViewRowValue    = padding: 0 0 @SkinWrapperPadding@ 0;
+DLEditViewTabPageTitle      = @FontSize-slightly-bigger@; @Text@; text-transform:none; line-height:1; padding: 0 0 @SkinWrapperPadding@ 0; margin:0; font-weight:normal;
+DLPropertiesView            = margin-top:@ContactViewGutterSpace@;
+DLPropertiesViewRowLabel    = @GroupEditViewFontSize@; vertical-align:top; padding:@SkinViewInnerSpacing@ 0 0 0;
+DLPropertiesViewRowValue    = padding: 0 0 @SkinWrapperPadding@ 0; @GroupEditViewFontSize@;
+DLPropertiesViewInput       = width:45em; margin:0;
```

Comment:
Yes, it was already there before and has been moved from inline styles to CSS rules. And width of `45em` seems good in this case.